src/main/java/view/NavigationPane.java

```diff
@@ -16,11 +31,24 @@ public static JPanel createNavigationPane(JPanel homepage) {
 
         JButton[] categoryButtons = {
                 new JButton("All Posts"),
-                new JButton("Category 1"),
+                new JButton("Java"),
```

Comment:
Everything looks good! I thought we could add categories that were a bit descriptive since that's how the categories were set up in my createpostview, I put sports, random, education and politics. If not, I can change my code to match this

Reply:
Sure I can add that! Are those the exact string names for the categories? (is it “Sports” or “sports” (lowercase)) etc? Since I need the names to match exactly for correct filtering, I need to know the if any of the categories are upper/lower case, singular/plural etc.

@izabellemarianne